app/lib/submission_builder/ty2024/states/nj/nj_return_xml.rb

```diff
@@ -26,6 +26,12 @@ def state_schema_version
             "NJIndividual2024V0.1"
           end
 
+          def build_state_specific_tags(document)
```

Comment:
[dust] great pattern introduced here, might be reusable for https://github.com/newjersey/affordability-pm/issues/230